pwa-devdocs/src/tutorials/pwa-studio-fundamentals/css-modules/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: CSS Modules
+---
+
+If you used the scaffolding tool to set up your PWA Studio storefront project, then [CSS Modules][] are supported out of the box without additional configuration.
+
+This tutorial provides some guidance on using CSS modules in your components.
+If you require more in depth information on CSS modules, see the [CSS Modules][] topic or [specification repository][].
+
+## Defining and using CSS modules
+
+CSS modules are CSS files which are imported and treated as JavaScript modules.
+
+The recommended naming convention for classes is camelCase naming because it looks cleaner when paired with dot notation(e.g. `style.className`).
+Kebab-case naming can produce unexpected behavior when accessing the class name using dot notation.
+Use bracket notation(e.g. `style['class-name']`) if you intend to use kebab-case.
+
+Example: `foo.css`
+
+```css
+.root {
+    padding: 3rem 1rem 1rem;
+    text-align: center;
+}
+
+.title {
+    text-transform: uppercase;
+}
+
+.title,
+.greeting {
+    padding-bottom: .5rem;
+}
+
+.spacer {
+    max-width: 500px;
+    border-color: #ddd;
+    border-top: 0;
+    margin: 36px auto;
+}
+
+.label {
+    color: #666;
+    padding-bottom: 8px;
+    font-style: italic;
+    font-size: 14px;
+}
+```
+
+Import CSS files as you would a JavaScript module and use dot or bracket notation to access the file name.
+
+Example: `foo.js`
+
+```jsx
+import React from 'react';
+
+import classes from './foo.css';
+
+const Foo = props => {
+    return (
+    <div className={classes.root}>
+        <h1 className={classes.title}>Foo Component</h1>
+        <hr className={classes.spacer} />
+        <p className={classes.label}>A child component with propTypes &amp; CSS Modules:</p>
+        <Greeting name="Joe Bloggs" className={classes.title} />
+    </div>
+    );
+}
+
+export default Foo;
+```
+
+## CSS modules in Venia UI components
+
+Venia UI components let you pass in a `classes` prop with a specific [prop type shape][] to override the class names used to render its elements.
+
+Example: `myButtonWrapper.js`
+
+```jsx
+import React from 'react';
+
+import Button from '@magento/venia-ui/lib/components/Button';
+
+import buttonOverrides from './myButtonWrapper.css';
+
+const MyButtonWrapper = props => {
+    const {
+        content,
+        root,
+        root_highPriority,
+        root_lowPriority,
+        root_normalPriority
+    } = buttonOverrides;
```

Comment:
Seems odd to destructure here only to create the exact same object shape below, and with long hand. A better example might be to show the contents of `myButtonWrapper.css`, and pass it directly without destructuring, which might be more effort than it's worth (maybe don't mock up fake properties, just show those class names exist in that file).

Another reason to destructure would be when names are not exactly the same, but that's not the case here.